data/ui_scripts/server_list/serverlist.lua

```diff
@@ -254,10 +681,129 @@ function menu_systemlink_join(f19_arg0, f19_arg1)
 	}, nil, nil, true)
 
 	menu:AddBackButton()
-
 	Lobby.RefreshServerList(Engine.GetFirstActiveController())
 
 	return menu
 end
 
+function showOnlyPublicServers()
```

Comment:
Variable names should be all lowercase

Reply:
Thats a function. Im not getting the issue

Reply:
The name has to be lowercase

Reply:
So, also showOnlyFavouritesServers, showOnlyBlacklistedServers?

Reply:
camel case and lowercase, show_only_favorites()